resources/lib/api.py

```diff
@@ -30,6 +31,11 @@ def magazines(lang):
     return _load_json(url).get('magazines', {})
 
 
+def home_categories(name, lang):
+    url = _endpoints['home'].format(category_code='home', lang=lang)
```

Comment:
`category_code` not needed

or you could refactor the `_endpoints` dict to merge similar paths (category & home) like so:

```python
_endpoints = {
    'categories': '/EMAC/teasers/{type}/v2/{lang}',
    'category': '/EMAC/teasers/category/v2/{category_code}/{lang}',
    'subcategory': '/OPA/v3/videos/subcategory/{sub_category_code}/page/1/limit/100/{lang}',
    'magazines': '/OPA/v3/magazines/{lang}',
    'collection': '/OPA/v3/videos/collection/{kind}/{collection_id}/{lang}',
    'streams': '/OPA/v3/streams/{program_id}/{kind}/{lang}',
    'daily': '/OPA/v3/programs/{date}/{lang}'
}
```

and api functions like

```python
def categories(lang):
    url = _endpoints['categories'].format(type='categories', lang=lang)
    return _load_json(url).get('categories', {})


def home_category(name, lang):
    url = _endpoints['categories'].format(type='home', lang=lang)
    return _load_json(url).get('teasers', {}).get(name, [])
```